### ISSUE TITLE
Accordions lack keyboard navigation

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus-bootstrap.trimble.com#138]
### Describe the issue

Accordions lack keyboard navigation. They should be tabbable and pressing enter should expand/collapse like Bootstrap 5 does:

https://pp3uik--run.stackblitz.io/

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

**Comment by coliff** [comment]
This is fixed in Modus Bootstrap v2.
